stories/module-search-console-settings.stories.js

```diff
@@ -24,51 +24,72 @@ import { storiesOf } from '@storybook/react';
 /**
  * Internal dependencies
  */
-import { STORE_NAME } from '../assets/js/modules/search-console/datastore/constants';
+import { MODULES_SEARCH_CONSOLE } from '../assets/js/modules/search-console/datastore/constants';
+import createLegacySettingsWrapper from './utils/create-legacy-settings-wrapper';
 import {
 	createTestRegistry,
 	provideUserAuthentication,
 	provideModules,
 	provideModuleRegistrations,
+	freezeFetch,
 } from '../tests/js/utils';
-import createLegacySettingsWrapper from './utils/create-legacy-settings-wrapper';
 
 const Settings = createLegacySettingsWrapper( 'search-console' );
 
 const defaultSettings = {
 	propertyID: '',
 };
 
-const withRegistry = ( Story ) => {
-	const registry = createTestRegistry();
-	registry.dispatch( STORE_NAME ).receiveGetSettings( {} );
-	provideUserAuthentication( registry );
-	provideModules( registry );
-	provideModuleRegistrations( registry );
+const storyOptions = {
+	decorators: [
+		( Story ) => {
+			const registry = createTestRegistry();
+
+			registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetSettings( {} );
 
-	return (
-		<Story registry={ registry } />
-	);
+			provideUserAuthentication( registry );
+			provideModules( registry );
+			provideModuleRegistrations( registry );
+
+			return <Story registry={ registry } />;
+		},
+	],
 };
 
 storiesOf( 'Search Console Module/Settings', module )
 	.add( 'View, closed', ( args, { registry } ) => {
 		return <Settings isOpen={ false } registry={ registry } />;
-	}, {
-		decorators: [
-			withRegistry,
-		],
-	} )
+	}, storyOptions )
 	.add( 'View, open with all settings', ( args, { registry } ) => {
-		registry.dispatch( STORE_NAME ).receiveGetSettings( {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetSettings( {
+			...defaultSettings,
+			propertyID: 'http://example.com/',
+		} );
+
+		return <Settings isOpen registry={ registry } />;
+	}, storyOptions )
+	.add( 'Edit, Loading', ( args, { registry } ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetSettings( defaultSettings );
+		freezeFetch( /^\/google-site-kit\/v1\/modules\/search-console\/data\/matched-sites/ );
+
+		return <Settings isOpen isEditing registry={ registry } />;
+	}, storyOptions )
+	.add( 'Edit, with all settings', ( args, { registry } ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetSettings( {
 			...defaultSettings,
 			propertyID: 'http://example.com/',
 		} );
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetMatchedProperties( [
+			{
+				permissionLevel: 'siteFullUser',
+				siteURL: 'https://www.example.io/',
```

Comment:
```suggestion
				siteURL: 'https://www.example.com/',
```

Reply:
It was intentionally added with a different TLD.